android/src/main/java/com/rnfingerprint/FingerprintHandler.java

```diff
@@ -5,63 +5,70 @@
 import android.os.Build;
```

Comment:
Why is there so many changes in this file? Due to indentation changes it's too hard to perform proper code review.

Reply:
I noticed I had a different Indentation. Now should be solved. Please, get back to me if is there any other mistake